openfisca_france_local/regions/nouvelle-aquitaine/aide_permis.py

```diff
@@ -0,0 +1,77 @@
+
+from openfisca_france.model.base import (
+    Variable, Individu, MONTH, TypesActivite, select, not_)
+from openfisca_france.model.caracteristiques_socio_demographiques.logement import (TypesCodeInseeRegion)
+from openfisca_france.model.prestations.education import TypesScolarite, TypesClasse
+
+
+class eligibilite_age_nouvelle_aquitaine_aide_permis(Variable):
+    value_type = float
+    entity = Individu
+    definition_period = MONTH
+
+    def formula(individu, period, parameters):
+        modalites_age = parameters(period).regions.nouvelle_aquitaine.aide_permis.age
+        age = individu('age', period)
+        age_minimum = modalites_age.minimum_inclusif
+        age_maximum = modalites_age.maximum_inclusif
+        return (age >= age_minimum) * (age <= age_maximum)
+    
+
+class eligibilite_profile_nouvelle_aquitaine_aide_permis(Variable):
+    value_type = float
+    entity = Individu
+    definition_period = MONTH
+
+    def formula(individu, period):
+        activite = individu('activite', period)
+        est_chomeur = activite == TypesActivite.chomeur
+
+        est_lyceen = individu('scolarite', period) == TypesScolarite.lycee
```

Comment:
Pour le même type de calcul tu assigne la variable `activité` vs pas la variable scolarité, je propose d'homogénéiser : 
```suggestion
        est_chomeur = individu('activite', period) == TypesActivite.chomeur
        est_lyceen = individu('scolarite', period) == TypesScolarite.lycee
```

Reply:
Très juste, bien vu et merci :)